scraper/twitter/utils.py

```diff
@@ -0,0 +1,45 @@
+
+
+def rate_tweet(tweet):
+    """
+    This method will return a coeficient for a tweet
+    following some rules
+
+    # rules
+    # - It should not be a reply
+    # - Each parameter will have a specific coefficient
+    # - We are going to have two arrays for 'fr' and 'en' maximum per array is 3 and minimum is 0
+    # Should not be sensitive or (empty value) parameter possibly_sensitive
+    # The same user can not have more than 2 tweets on each array of tweets
+    """
+
+    FAVORITE_COEFFICIENT = 0.3
+    RETWEET_COEFFICIENT = 0.2
+    IS_NOT_REPLY_COEF = 0.15
+    IS_QUOTED_COEF = 0.09
+    # We allocate coefs for each lang
+    LANG_COEF = {"en": 0.04, "fr": 0.021, "und": 0.015}
+    TEXT_SIZE_COEF = 0.07
+
+    coef = 0
+
+    if tweet["possibly_sensitive"] is not True:
+        coef = tweet["favorite_count"] * FAVORITE_COEFFICIENT
+        coef += tweet["retweet_count"] * RETWEET_COEFFICIENT
+
+        # lang coef
+        if "fr" in tweet["lang"] or "en" in tweet["lang"] or "und" in tweet["lang"]:
+            coef += LANG_COEF[tweet["lang"]]
+
+        # We add to the coeficient  the coef for the text size
+        coef += len(str(tweet["text"]).replace("#CaParleDev")) * TEXT_SIZE_COEF
```

Comment:
Note that sometime users on twitter write #CaParleDev or #caparledev or #caparleDev, etc...
This is why i suggest to have a function _get_tag(tweet) that will retrieve CaParleDev tag writen in different ways in the tweet and then return the proper tag that will be CaParleDev


For example:
```py
def _get_tag(tweet):
    tags = ["#CaParleDev", "#caparledev", etc....]
    for t in tags:
        if t in tweet:
            return "#CaParleDev"
    return False
```

Then, this line would be something like that:

```py
if _get_tag(tweet):
    coef += len(str(tweet["text"]).replace(_get_tag(tweet))) * TEXT_SIZE_COEF
```

You can also improve this